lahja/endpoint.py

```diff
@@ -420,15 +488,18 @@ def subscribe(self,
             self._queues[event_type] = []
 
         self._queues[event_type].append(queue)
+        self._notify_subscriptions_changed()
         i = None if num_events is None else 0
         while True:
             try:
                 yield await queue.get()
             except GeneratorExit:
                 self._queues[event_type].remove(queue)
+                self._notify_subscriptions_changed()
```

Comment:
Maybe a small cleanup helper function grouping these two repeated lines would be justified now.